include/simics/dmllib.h

```diff
@@ -1780,6 +1780,11 @@ _get_port_saved_variable(lang_void *saved_access, conf_object_t *_portobj,
                                   acc->getter);
 }
 
+
+UNUSED static uint64 _identity_to_key(_identity_t id) {
+        return id.id << 32 || id.encoded_index;
```

Comment:
`|` not `||`. I also suspect the implicit types involved are incorrect: that should be fixed by this:
```
return (uint64)id.id << 32 | id.encoded_index;
```

Reply:
Yeah I found this out too, felt pretty stupid.